### ISSUE TITLE
Screen Reader announces content page tab as "document unknown"

### ISSUE THREAD

**Issue by Shobhit1** [Microsoft/accessibility-insights-web#644]
## Describe the bug

Content reader (NVDA) announces the new tab opened for content

**To Reproduce**

1. Open any assessment from Assessment option on AI4W browser extension
2. Open NVDA
3. ab through to the Guidance link and press Space
4. Check NVDA output

## Expected behavior

NVDA reads  `document unknown` before announcing the title of the content page.
It should just announce the title and not `document unknown`.

## Screenshots

![image](https://user-images.githubusercontent.com/4496335/57045084-6f632580-6c21-11e9-8720-33d5a9645f3e.png)

## Extension (please complete the following information)

- OS: Windows 10
- Version: Insider 
- Browser Version: Edge Insider Version 75.0.139.1 (Official build) dev (64-bit)
- Target Page: Doesn't matter

## Are you willing to submit a PR?

Yes

**Comment by ferBonnin** [comment]
@iamrafan thoughts on this?

**Comment by iamrafan** [comment]
@ferBonnin NVDA is doing this because of the ```target="_blank"``` attribute on the hyperlink used to navigate to this page. Let me file it with NVDA team and find out their take on the issue

**Comment by iamrafan** [comment]
Filed https://github.com/nvaccess/nvda/issues/9533

**Comment by ferBonnin** [comment]
thanks @iamrafan!

**Comment by ferBonnin** [comment]
per reply on NVDA bug this seems like a weird edge condition. @Shobhit1 does this repros if you start NVDA before?

**Comment by Shobhit1** [comment]
@ferBonnin  For this I have NVDA running on my machine, before I clicked on any other link.  I dont completely understand what "starting NVDA before" means. Sorry.

**Comment by ferBonnin** [comment]
@Shobhit1 repro steps indicate open Assessment and then launch NVDA, per the bug Guru opened [nvaccess/nvda#9533](https://github.com/nvaccess/nvda/issues/9533) the workflow seems a bit odd.

**Comment by iamrafan** [comment]
@ferBonnin I don't think it has anything to do with when NVDA is started, I think it related to focus and how NVDA handles pages without a focusable component

**Comment by karanbirsingh** [comment]
@iamrafan @ferBonnin looks like there's a response on the bug Guru filed - we might want to reply

**Comment by DaveTryon** [comment]
After checking with Karan, the status of this should not have been changed on 9 October. Undoing those changes.

**Comment by ferBonnin** [comment]
@iamrafan friendly ping to take a look at the reply of the bug you filed, please

**Comment by iamrafan** [comment]
@Shobhit1 I do not see this issue anymore, can you please let me know if you are able to repro?
